src/opinf/models/mono/_nonparametric.py

```diff
@@ -266,7 +267,9 @@ def refit(self):
             return self
 
         # Execute non-intrusive learning.
-        self._extract_operators(self.solver.solve())
+        self._Ohat = self.solver.solve()
```

Comment:
This is already implemented in the `operator_matrix` attribute, but not a big deal